scripts/getRedstonePayload.log.txt

```diff
@@ -0,0 +1,2 @@
+You have to provide a data FeedAn error occurred: Request failed {"reqParams":{"dataServiceId":"redstone-primary-prod","uniqueSignersCount":3,"dataFeeds":["--no-warnings"],"urls":["https://oracle-gateway-1.a.redstone.finance"]}}, Original error: AggregateError: , errors: Error: Requested data feed id is not included in response: --no-warnings
```

Comment:
This should not be committed and given its in the .gitignore it wont be in the future.